charts/consul/templates/crd-httprouteauthfilters.yaml

```diff
@@ -0,0 +1,140 @@
+{{- if .Values.connectInject.enabled }}
+---
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  annotations:
+    controller-gen.kubebuilder.io/version: v0.11.3
+  creationTimestamp: null
+  name: httprouteauthfilters.consul.hashicorp.com
+  labels:
+    app: {{ template "consul.name" . }}
+    chart: {{ template "consul.chart" . }}
+    heritage: {{ .Release.Service }}
+    release: {{ .Release.Name }}
+    component: crd
+spec:
+  group: consul.hashicorp.com
+  names:
+    kind: HTTPRouteAuthFilter
```

Comment:
It seems like this should be `RouteAuthFilter` to stay consistent with some shifts in Sarah's PR for timeout

Reply:
so in the spec GrpcRoute filters just reuse the `HTTP` prefixed filters: https://gateway-api.sigs.k8s.io/references/spec/#gateway.networking.k8s.io/v1alpha2.GRPCRouteFilter
should we follow suit there or be internally consistent?

Reply:
Being internally consistent is the more important thing IMO since these are CRDs.
If y'all want to go back and change `RouteRetryFilter` => `HTTPRouteRetryFilter`, then I'm fine with that. I believe @andrewstucki had stronger feelings about that than I do.

At the end of the day, I care most about internal consistency. Whether that's `Route...` or `HTTPRoute...` is fine with me.

Reply:
either works for me, less typing for our end users is probably better

Reply:
Discussed with the team and landed on `RouteAuthFilter`